core/api/src/app/callback/index.ts

```diff
@@ -54,3 +54,11 @@ export const deleteEndpoint = async ({
   if (success instanceof Error) return success
   return success
 }
+
+export const getPortal = async (accountId: AccountId) => {
```

Comment:
should not be `getPortalUrl`? also please use `{accountId}: {accountId: AccountId}` this creates `code.function.params.accountId` in span attrs

Reply:
oh I didn't know that.